django-stubs/contrib/gis/gdal/datasource.pyi

```diff
@@ -1,16 +1,18 @@
 from typing import Any
 
 from django.contrib.gis.gdal.base import GDALBase as GDALBase
+from django.contrib.gis.gdal.driver import Driver as Driver
+from django.contrib.gis.gdal.layer import Layer as Layer
```

Comment:
These probably shouldn't be re-exported. The `GDALBase` above (which you didn't touch, to be fair) looks wrong too.

```suggestion
from django.contrib.gis.gdal.base import GDALBase
from django.contrib.gis.gdal.driver import Driver
from django.contrib.gis.gdal.layer import Layer
```